### ISSUE TITLE
roadmap?

### ISSUE THREAD

**Issue by cyberhck** [router5/router5#483]
Just curious what's next for router5, is it being maintained? Is it feature frozen? Are you working on something else?

**Comment by iangregsondev** [comment]
Also wondering about this, I see a lot of pull requests open but no activity.

I was really want to use this to see if i could support routing on a state library.

**Comment by aleksandrlat** [comment]
@troch could you please transfer this repository to somebody else if you don't have time to maintain this project anymore?

**Comment by greydragon888** [comment]
Yes! I have the roadmap: https://github.com/greydragon888/real-router/issues. 
This list is not full. I will extend it.
Join to contribution!
